docs/developer-guide.rst

```diff
@@ -128,6 +128,7 @@ and upload a file:
 
     audbackend.backend.FileSystem.create("./host", "repo")
     backend = audbackend.backend.FileSystem("./host", "repo")
+    backend.open()
```

Comment:
I think for good practice we should call `close()` before we delete the repository.

Reply:
I added:

![image](https://github.com/audeering/audbackend/assets/173624/34b62b23-6a65-45a7-a059-66205197196b)